app/javascript/packs/runner_application.js

```diff
@@ -8,9 +8,9 @@ require("@rails/ujs").start()
 const accessibleAutocomplete = require("accessible-autocomplete")
 import 'accessible-autocomplete/dist/accessible-autocomplete.min.css' 
```

Comment:
Really bugs me we have to do this inside a JS file. Sure there must be a better way to setup the webpack stuff so that JS and CSS are separate :-( 

(no, not saying 'change it', just having a little whinge)